lib/python/qmk/search.py

```diff
@@ -61,10 +61,22 @@ def apply(self, target_info: TargetInfo) -> bool:
 
 class Length(FilterFunction):
     func_name = "length"
+    funcMap = [
```

Comment:
Just a nitpick, camelCase looks off when everything else is snake_case... Its the "official" format for vars anyway.